parlai/crowdsourcing/tasks/fast_acute/README.md

```diff
@@ -0,0 +1,99 @@
+# Fast ACUTE
```

Comment:
I notice this README does not include being able to run fast acute with tasks/datasets, is that still supported? i.e. are there things mentioned in https://github.com/facebookresearch/ParlAI/tree/master/parlai/mturk/tasks/acute_eval that are not available here?

Reply:
Discussed offline - yeah, that's the only Fast ACUTE function that I know of left to add. I'll hold off on removing the old version until that's added